.circleci/config.yml

```diff
@@ -5,8 +5,8 @@ aliases:
     context: spaceshard
     filters:
       branches:
-        only:
-          - master
+        ignore:
+          - /^.*$/
```

Comment:
Why are we ignoring the workflow on all branches?

Reply:
https://github.com/0xSpaceShard/cairo-cli-docker/pull/18/commits/c0b97d630d63ce2722d8753674165bd184070b9f